addons/event/models/event.py

```diff
@@ -42,6 +43,7 @@ def _get_default_event_type_mail_ids(self):
         })]
 
     name = fields.Char('Event Category', required=True, translate=True)
+    sequence = fields.Integer('sequence')
```

Comment:
No need for the `sequence` string, actually it is even wrong as it would not have an uppercase starting letter.